wsdl/ver20/analytics/humanface.xsd

```diff
@@ -449,9 +449,9 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				<xs:documentation>Describe the eye of the face. </xs:documentation> 
 			</xs:annotation> 
 			</xs:element> 			
-			<xs:element name="Ear" type="fc:Ear" minOccurs="0"> 
+			<xs:element name="Ear" type="xs:string" minOccurs="0"> 
```

Comment:
changing Ear element data type to xs:string from fc:Ear makes ONVIF device needs to validate allowed string values for Ear in the  application source code against fc:EarShape values. But previously, the data type fc:Ear makes restriction on string enumerations in the generated gsoap stub code in the device side with out extra validation in application source code. I understand this change will not create interoperability issue.